contracts/ServiceNodeContribution.sol

```diff
@@ -111,12 +115,31 @@ contract ServiceNodeContribution is Shared {
      */
     function contributeOperatorFunds(
         uint256 amount,
-        IServiceNodeRewards.BLSSignatureParams memory _blsSignature
+        IServiceNodeRewards.BLSSignatureParams memory _blsSignature,
+        IServiceNodeRewards.Contributor[] memory _reservedContributors
```

Comment:
Does solidity allow a `= []` default value for this argument so that you can just call it without the reserved arguments?

Reply:
AFAIK it doesnt - they have a discussion here which implies it is not planned either https://github.com/ethereum/solidity/issues/232

Reply:
No worries then.  It does support overloading, so I suppose we could have a second version with the same name without reserved contributions, but it's probably not worthwhile versus just requiring an empty array here.